tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testenum-testenumvalue1-enummember.html

```diff
@@ -1,33 +1,20 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        TestEnumValue1
-      </h2>
+      <h2>TestEnumValue1</h2>
       <section>
-        <p>
-          Test enum value 1 (string)
-        </p>
+        <p>Test enum value 1 (string)</p>
       </section>
       <section>
-        <h3 id="testenumvalue1-signature">
-          Signature
-        </h3>
-        <code>
-          TestEnumValue1 = &quot;test-enum-value-1&quot;
-        </code>
+        <h3 id="testenumvalue1-signature">Signature</h3><code>TestEnumValue1 = "test-enum-value-1"</code>
```

Comment:
I think going from `&quot;` to `"` is fine as long as we're in the contents of an HTML element... so this has no impact? But might be worth double checking.

Reply:
Yeah, `"` is valid when it isn't within the context of a tag declaration. Confirmed that this is rendering correctly on the website (with these changes applied).

![image](https://github.com/user-attachments/assets/797a86f8-ca4d-46a0-b872-4889baff16e1)